test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -2281,8 +2315,8 @@ def re_compile(value):
 
 def run():
     parser = argparse.ArgumentParser(prog="validate-modules")
-    parser.add_argument('modules', nargs='+',
```

Comment:
I'm not sure if anyone has ever used these options. 
Do we need to keep an (undocumented) `--modules` for backwards compatability?

Reply:
`modules` is in fact just the variable the positional arguments are stored in (and maybe it's also used as the default placeholder string in the help), so the CLI interface is already backwards compatible.